scripts/purge_queue.py

```diff
@@ -0,0 +1,83 @@
+import ast
+import sys
+
+import argparse
+import boto3
+
+from azul import config
+
+parser = argparse.ArgumentParser(description=__doc__)
+
+subparsers = parser.add_subparsers(help="sub-command help")
+
+parser_dump = subparsers.add_parser("dump", help="Dump contents of queue into designated file")
+parser_dump.add_argument("queue_name", help="Name of the queue to obtain the messages from")
+parser_dump.add_argument("file_name", help="Name of file to direct output")
+parser_dump.add_argument("--delete", "-d", action='store_true', help="Purge messages from the queue")
+
+parser_list = parser.add_argument("--list", "-l", help="List all available queues", action="store_true")
+
+parser_json_body = parser.add_argument("--json-body", "-jb", help="Output messages in JSON format", action="store_true")
+
+
+def get_messages_from_queue(args):
+    global output_file_name, queue_name
+    sqs = boto3.resource("sqs")
+
+    if args.list:
+        prefix = config._index_prefix  # Limit queues displayed to our deployment prefix
+        for queue_item in list(filter(lambda queue_url: prefix in queue_url, [q.url for q in sqs.queues.all()])):
```

Comment:
`queue_item` is very misleading naming.

Reply:
`prefix in s` is ridiculous. If it's a prefix it should be `s.startswith(prefix)`